client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/coverage/methodreplacement/classes/InetAddressClassReplacement.java

```diff
@@ -0,0 +1,66 @@
+package org.evomaster.client.java.instrumentation.coverage.methodreplacement.classes;
+
+import org.evomaster.client.java.instrumentation.ExternalServiceInfo;
+import org.evomaster.client.java.instrumentation.coverage.methodreplacement.MethodReplacementClass;
+import org.evomaster.client.java.instrumentation.coverage.methodreplacement.Replacement;
+import org.evomaster.client.java.instrumentation.coverage.methodreplacement.UsageFilter;
+import org.evomaster.client.java.instrumentation.shared.ReplacementCategory;
+import org.evomaster.client.java.instrumentation.shared.ReplacementType;
+import org.evomaster.client.java.instrumentation.staticstate.ExecutionTracer;
+
+import java.net.*;
+
+public class InetAddressClassReplacement implements MethodReplacementClass {
```

Comment:
add comment that this class is not in used, eg not in `ReplacementList`, but kept if in future we want to mock it